src/mobile/apps/search/client.coffee

```diff
@@ -17,8 +17,6 @@ module.exports.SearchResultsView = class SearchResultsView extends Backbone.View
         if result.display_model is 'artwork'
           @refreshRenderArtworks result
 
-    splitTest('new_search_page').view()
```

Comment:
The import of `splitTest` on [line 6](https://github.com/artsy/force/pull/3995/files#diff-40111ebee2292fc5f1aab1e232858539R6) can also be removed, now that we're no longer using it.